src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -0,0 +1,71 @@
+from django.conf import settings
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint
+from sentry.api.permissions import SuperuserPermission
+
+# from sentry.api.serializers import SimpleEventSerializer, serialize
+from sentry.eventstream.kafka import KafkaEventStream
+from sentry.utils import json
+
+
+class IssueOccurrenceSerializer(serializers.Serializer):
+    # id = serializers.IntegerField()
+    event_id = serializers.CharField()
+    fingerprint = serializers.ListField()
+    issue_title = serializers.CharField()
+    subtitle = serializers.CharField()
+    evidence_data = serializers.DictField()
+    evidence_display = serializers.ListField()
+    type = serializers.IntegerField()
+    detection_time = serializers.DateTimeField()
+
+
+class IssueOccurrenceEndpoint(Endpoint):
+    permission_classes = (SuperuserPermission,)
+    private: True
+
+    def post(self, request: Request) -> Response:
+        """
+        Write Issue occurrence data to a Kafka topic
+        `````````````````````````````````
+        :auth: required
+        """
+        event = request.data.pop("event")
+        occurrence = request.data
+
+        # this needs to be passed an Event or GroupEvent instance which obviously doesn't work here
```

Comment:
What if we accepted an event ID instead and looked it up? I'm not actually sure what we're accomplishing by sending raw event data here, but I'm not sure how to best handle it since we don't typically do this through the API.

Reply:
You don't need to be passed an actual `Event`, you just need a dict that matches the schema.

Based on https://www.notion.so/sentry/Issue-Platform-Problem-Ingestion-18372cf235314321b15779025f72d532, you could just do a bare minimum event that has `event_id`, `title`, `platform`, `tags` (could be an empty dict), `timestamp`, `message_timestamp` (tbh we could hack this in your endpoint to just be timestamp as well)`

The reason we need raw event data is that there are no events for profiles, they just have a totally separate data source. So we're asking them to send us data that matches the event format, and we're storing out own events as part of the platform.